index.html

```diff
@@ -108,6 +108,16 @@ <h3>Concept and Initial Report</h3>
       <div class="city-details-text"></div>
     </div>
 
+    <form id="filter">
+      <select multiple class="scope-filter">
```

Comment:
Probably a better naming scheme is `filter-scope`, i.e. `filter-x`. It's nice when everything has the same prefix because it makes it more obvious what is related.

You could also consider `filter--scope` (`filter--x`)